samples/qa-a11y/src/views/MediaPlayer.js

```diff
@@ -0,0 +1,41 @@
+import MediaPlayer from '@enact/agate/MediaPlayer';
+import React from 'react';
+
+import Section from '../components/Section';
+
+import appCss from '../App/App.module.less';
+
+const audioFiles = [
+	'https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3',
+	'https://sampleswap.org/mp3/artist/78152/HiatusManJBanner_Show-Stopper-160.mp3',
+	'https://sampleswap.org/mp3/artist/47067/DJ-Masque_Oceanic-Dawn-160.mp3',
+	'https://sampleswap.org/mp3/artist/26546/benzoul_lovevoodoo-160.mp3',
+	'https://sampleswap.org/mp3/artist/19139/MarkNine_In-my-Place-160.mp3',
+	'https://sampleswap.org/mp3/artist/47067/DJ-Masque_Dont-Forget-To-Be-Yourself-160.mp3'
+];
+
+const source = audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />));
+
+const MediaPlayerView = () => (
+	<>
+		<Section title="Default">
+			<MediaPlayer alt="Normal">
```

Comment:
MediaPlayer seems too right in the sample.

Reply:
I hope we could go ahead.

Reply:
Let's remove alt..